verification/curator-service/api/test/cases.test.ts

```diff
@@ -27,6 +31,11 @@ afterAll(() => {
     return mongoose.disconnect();
 });
 
+afterEach(async () => {
```

Comment:
Should this be run before all as well to ensure the tests start out with a clean slate? Same for next test

Reply:
it's a tradeoff between "starting with a clean state and ending with a clean state" ;) I agree starting with a clean state sounds better though. Done.